Makefile

```diff
@@ -300,10 +301,8 @@ install: build
 ###### TARGETS FOR CICD PROCESS ######
 include ./scripts/release/mule/Makefile.mule
 
-SUPPORTED_ARCHIVE_OS_ARCH = linux/amd64 linux/arm64 linux/arm darwin/amd64
-
 archive:
 	CHANNEL=$(CHANNEL) \
-	PATH=$(SRCPATH)/tmp/node_pkgs/$(OS_TYPE)/$(ARCH)/bin:$${PATH} \
-	scripts/upload_version.sh $(CHANNEL) $(SRCPATH)/tmp/node_pkgs $(S3_RELEASE_BUCKET)
+	FULLBUILDNUMBER=$(FULLBUILDNUMBER) \
+	aws s3 cp tmp/node_pkgs s3://algorand-internal/channel/${CHANNEL}/${FULLBUILDNUMBER} --recursive --exclude "*" --include "*${CHANNEL}*${FULLBUILDNUMBER}*"
```

Comment:
Interesting, just to be clear here, this is uploading to algorand-internal, not algorand-staging, and will do so to a subdirectory based on the buildnumber? That should be fine, because it won't overlap with Travis's build artifacts if that's the case (uploads to the root of that dir).